participants.md

```diff
@@ -2,79 +2,113 @@
+  - {name: "Mike Ogiugo", affiliation: "Yaba College of Technology", country: "Nigeria", country_emoji: "🇳🇬", online: false}
+  - {name: "Milad Ahanjideh", affiliation: "University of Primorska", country: "Slovenia", country_emoji: "🇸🇮", online: false}
+  - {name: "Morgan Rodgers", affiliation: "RPTU Kaiserslautern-Landau University", country: "Germany", country_emoji: "🇩🇪", online: false}
+  - {name: "Neda Ahanjideh", affiliation: "Shahrekord University", country: "Iran", country_emoji: "🇮🇷", online: false}
+  - {name: "Nora Harrach", affiliation: "Eötvös Loránd University", country: "Hungary", country_emoji: "🇭🇺", online: false}
+  - {name: "Olexandr Konovalov", affiliation: "University of St Andrews", country: "United Kingdom", country_emoji: "🇬🇧", online: false}
+  - {name: "Óscar Fernández Ayala", affiliation: "TU Braunschweig", country: "Germany", country_emoji: "🇩🇪", online: false}
+  - {name: "Rhys Evans", affiliation: "IMFM", country: "Slovenia", country_emoji: "🇸🇮", online: false}
+  - {name: "Russ Woodroofe", affiliation: "University of Primorska", country: "Slovenia", country_emoji: "🇸🇮", online: false}
+  - {name: "Sarah Rees", affiliation: "University of Newcastle", country: "United Kingdom", country_emoji: "🇬🇧", online: false}
+  - {name: "Seyyed Ali Mohammadiyeh", affiliation: "Department of Pure Mathematics, Faculty of Mathematical Sciences, University of Kashan", country: "Iran", country_emoji: "🇮🇷", online: false}
+  - {name: "Thomas Breuer", affiliation: "RWTH Aachen University", country: "Germany", country_emoji: "🇩🇪", online: false}
+  - {name: "Timo Velten", affiliation: "TU Braunschweig", country: "Germany", country_emoji: "🇩🇪", online: false}
+  - {name: "Vinay Wagh", affiliation: "IIT Guwahati", country: "India", country_emoji: "🇮🇳", online: false}
+  - {name: "Volkmar Welker", affiliation: "Philipps-Universität Marburg", country: "Germany", country_emoji: "🇩🇪", online: false}
 ---
 
-<ol>{% assign participants = page.participants %}
-{% for p in participants %}
+<ol>
+{% for p in page.participants %}
   <li>
     <strong>{{ p.name }}</strong>
-    {% if p.affiliation != null %} ({{ p.affiliation }}){% endif %}
-    {% if p.links != null %}
-        {% for item in p.links %}
-            <a href="{{ item[1] }}">({{ item[0] }})</a>
-        {% endfor %}
+    {% if p.affiliation %} ({{ p.affiliation }}){% endif %}
+    {% if p.country_emoji %} {{ p.country_emoji }}{% endif %}
+    {% if p.online %}
+      <span style="color: green;">[Online]</span>
+    {% endif %}
+    {% if p.links %}
+      {% for item in p.links %}
+        <a href="{{ item[1] }}">({{ item[0] }})</a>
+      {% endfor %}
     {% endif %}
     <br/>
-      {% if p.talk != null %} Talk: {{ p.talk }}{% endif %}
+    {% if p.talk %} Talk: {{ p.talk }}{% endif %}
   </li>
 {% endfor %}
 </ol>
 
-{% if site.data.feedback.size > 0 %}
+<h2>Participants by Country ({{ page.participants | size }})</h2>
```

Comment:
While this is a fun statistic, I don't think we should show this on the participants page in general.